src/js/components/Navigation/BallotFilter.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { Component, PropTypes } from "react";
+import { Link } from "react-router";
+
+export default class BallotFilter extends Component {
+  static propTypes = {
+    params: PropTypes.object,
+    ballot_type: PropTypes.string
+  };
+
+
+  render () {
+    const {ballot_type} = this.props;
+
+    return <div className="btn-group">
+      <Link to="/ballot" className={ ballot_type === "ALL_BALLOT_ITEMS" ? "active btn btn-default" : "btn btn-default"}>
```

Comment:
Find a better way to add `active` as a conditional class for selected nav items.